### ISSUE TITLE
Add Solaris/x64 as a supported testing platform

### ISSUE THREAD

**Issue by sxa** [AdoptOpenJDK/openjdk-tests#2060]
**Describe the bug** Unable to submit jobs to test our Solaris/X64 bulds as we do not have x86-64_solaris as a viable `PLATFORM` option in https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/buildenv/jenkins/openjdk_tests We do have `sparcv9_solaris` but not the X64 variant

**To Reproduce** Try to run a job (e.g. a Grinder) and it's not an option for `PLATFORM`

**Expected behavior** Ability to run tests against Solaris/x64 builds

**Screenshots**

**Additional context** JDK8 / HotSpot Release Builds for Solaris/x64 are at https://adoptopenjdk.net/releases.html?variant=openjdk8&jvmVariant=hotspot and the job that builds them is at https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk8u/job/jdk8u-solaris-x64-hotspot/

**Comment by smlambert** [comment]
This issue is gated by having test machines available, is there an infrastructure issue that we can watch, and once there are machines, we can move forward with enabling tests.  Note that I have already done many of the changes we think we need, just require machine(s) with test prereqs on them.

**Comment by sxa** [comment]
Is there a job we can use? I think last time I looked at it there weren't any set up that I could run. Happy to attempt initial runs on the build machines

**Comment by smlambert** [comment]
I have generated some sparcv9_solaris jobs [here](https://ci.adoptopenjdk.net/view/Test_grinder/job/Test_Job_Auto_Gen/234/)

Will generate x64-64_solaris jobs shortly

**Comment by smlambert** [comment]
Generated x64-64_solaris jobs [here](https://ci.adoptopenjdk.net/view/Test_grinder/job/Test_Job_Auto_Gen/254/)
